easybuild/easyconfigs/l/likwid/likwid-4.2.0-intel-config-mk.patch

```diff
@@ -0,0 +1,49 @@
+--- likwid-likwid-4.2.orig/config.mk	2016-12-22 13:22:23.000000000 +0100
++++ likwid-likwid-4.2/config.mk	2017-06-22 10:14:28.585696000 +0200
+@@ -1,7 +1,7 @@
+ # Please have a look in INSTALL and the WIKI for details on
+ # configuration options setup steps.
+ # supported: GCC, CLANG, ICC, MIC (ICC), GCCX86 (for 32bit systems)
+-COMPILER = GCC#NO SPACE
++COMPILER = ICC#NO SPACE
```

Comment:
That's the advised guidelines (from [INSTALL](https://github.com/RRZE-HPC/likwid/blob/master/INSTALL) as the `COMPILER` setting in `config.mk` permits to  `include_[GCC|CLANG|ICC|MIC].mk` which are required for correct compilation. 
I wouldn't risk to diverge from this approach. Unless you mean there is a way to reuse the foss patch and provide the appropriate `sed` command to be performed in addition to the patch yet _before_ the build ? That would be confusing for me.